scripts/utils/mineBlock.ts

```diff
@@ -0,0 +1,14 @@
+import * as helpers from '@nomicfoundation/hardhat-network-helpers';
+import { env } from '../../config/env';
+
+// This function is used to force mining if we are one a local fork
+export async function mineBlock() {
```

Comment:
We can make the function name more explicit and allow reading the comment as a JSDoc.
```suggestion
/**
 * This function is used to force mining if we are one a local fork
 */
export async function mineBlockIfOnLocalFork() {
```